tool/gravity/cli/ops.go

```diff
@@ -101,7 +101,9 @@ func uploadUpdate(env *localenv.LocalEnvironment, opsURL string) error {
 
 	clusterOperator, err := defaultEnv.SiteOperator()
 	if err != nil {
-		return trace.Wrap(err)
+		return trace.Wrap(err, "unable to access cluster.\n"+
```

Comment:
This is likely more user-friendly than the original `open /var/lib/gravity/local/.../kubectl.kubeconfig: file or directory not found`